api/api.proto

```diff
@@ -156,14 +156,11 @@ message BolusAmountMaximumGuardRail {
   // Bolus units
   BolusUnits units = 1;
 
-  // Default value
-  FixedDecimal defaultValue = 2;
-
   // Tidepool recommended bounds
-  RecommendedBounds recommendedBounds = 3;
+  RecommendedBounds recommendedBounds = 2;
```

Comment:
This change breaks backward compatibility of the binary protocol. It should be reverted.

Reply:
Reverted.